django/core/cache/backends/db.py

```diff
@@ -49,52 +49,78 @@ class DatabaseCache(BaseDatabaseCache):
     pickle_protocol = pickle.HIGHEST_PROTOCOL
 
     def get(self, key, default=None, version=None):
-        key = self.make_key(key, version=version)
-        self.validate_key(key)
```

Comment:
Where is this called in `get_many()`? Can you explain why it is not needed?

Reply:
@pope1ni I am still reviewing, but it looks like the `get_many()` function performs `validate_key()` on line 60, so they removed it here. I am still looking at `get_many` more closely, but I suspect it will work fine, though I personally think we should revert these two lines so that future developers do not have to read through the entire `get_many()` function to understand the only thing that is happening here is that `validate_key()` is being run.

Reply:
I see now that a lot more was going on here than I initially thought. I am coming around to being fine with your approach here @MyungSeKyo

Reply:
@jeffyancey - this comment is old and `validate_key()` was re-added after my review - it wasn't flagged as outdated because the line the comment is on wasn't changed.

Reply:
makes sense.